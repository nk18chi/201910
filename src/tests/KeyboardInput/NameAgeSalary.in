Dennis
37
8.50
