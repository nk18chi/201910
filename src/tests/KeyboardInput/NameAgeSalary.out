Hello. What is your name?

Hi, Dennis!  How old are you?

So you're 37, eh?  That's not old at all!
How much do you make, Dennis?

8.5!  I hope that's per hour and not per year! LOL!
